src/lib.rs

```diff
@@ -821,8 +827,9 @@ impl Config {
         cmd.arg("--config").arg(&profile);
 
         if let Ok(s) = env::var("NUM_JOBS") {
-            // See https://cmake.org/cmake/help/v3.12/manual/cmake.1.html#build-tool-mode
-            cmd.arg("--parallel").arg(s);
+            if cmake_has_parallel(&executable) {
+                cmd.arg("--parallel").arg(s);
+            }
```

Comment:
Could this add some sort of warning if `--parallel` isn't passed? I can imagine one day the version parsing fails for some reason and having a warning here might help debug a bit.

Reply:
Added a warning here, and another in the cmake_has_parallel function (I think it might be more appropriate there).